README.md

```diff
@@ -426,6 +428,23 @@ const { token } = await auth();
 
 See [@octokit/auth-action](https://github.com/octokit/auth-action.js#readme) for more details.
 
+## Community Strategies
+
+### .netrc authentication
+
+Similar to [token authentication](#token-authentication), but reads the token from your `~/.netrc` file
+
+Example
+
+```js
+// expects a personal access token to be set as `login` in the `~/.netrc` file for `api.github.com`
+const createNetrcAuth = require('octokit-netrc-auth').createNetrcAuth;
```

Comment:
I think I would use deconstruction here, as I'm using it in all Octokit examples

```suggestion
const { createNetrcAuth } = require('octokit-netrc-auth');
```

Reply:
Yep good call. I didn't pay close enough attention when I was trying to match.